jest.config.js

```diff
@@ -34,10 +40,10 @@ module.exports = {
   // picked up by jest if this folder is not excluded causing a false negative of a test suite failing.
   testPathIgnorePatterns: [
     '/node_modules/',
-    './config/webpack',
+    '<rootDir>/config/webpack',
```

Comment:
Interesting, I didn't know about this...variable? (Is that what this is?)

I suppose we should have been using this from the get-go! :)